sdk/eventhub/Azure.Messaging.EventHubs/Azure.Client.Library.References.props

```diff
@@ -0,0 +1,11 @@
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
One last nit, and apologies for not having thought of this sooner.    Could we move this file to the `samples` directory, as it is currently only applicable to the samples.

By contrast, the one in the Blobs CheckpointStore library is used across the project as a common point of reference.

Reply:
This is invalid;  it was pointed out to me that references to this are made by both tests and samples.   It's in the correct location.   My bad.